_includes/footer.html

```diff
@@ -1,28 +1,51 @@
-<footer class="footer bg-navy">
-  <div class="footer-nav bg-lightest-blue">
+<footer class="footer">
+  <div class="footer-nav">
     <div class="container">
+      <!-- apply to .footer-nav -->
       <div class="navigation">
         <nav>
           <div class="grid">
+            <!-- apply to nav -->
             <section>
+              <!-- replace with uls -->
               <h6>{% t nav.groups.agencies %}</h6>
-              <a class="h6" target="_blank" href="https://partners.login.gov">{% t nav.become_a_partner %}</a>
-              <a class="h6" target="_blank" href="https://developers.login.gov">{% t nav.developer_guide %}</a>
+              <a class="h6" target="_blank" href="https://partners.login.gov"
+                >{% t nav.become_a_partner %}</a
+              >
+              <a class="h6" target="_blank" href="https://developers.login.gov"
+                >{% t nav.developer_guide %}</a
+              >
             </section>
             <section>
               <h6>{% t nav.groups.learn %}</h6>
-              <a class="h6" href="{{ site.baseurl }}/about/">{% t nav.about_us %}</a>
-              <a class="h6" href="{{ site.baseurl }}/policy/">{% t nav.privacy_&_security %}</a>
-              <a class="h6" target="_blank" href="https://design.login.gov/accessibility">{% t nav.accessibility %}</a>
+              <a class="h6" href="{{ site.baseurl }}/about/"
```

Comment:
`h6` is a BassCSS class if I recall correctly. Wonder why that wasn't caught in the previous search pattern results I put together 🤔

Reply:
It is. I replaced it with a custom class.

Reply:
> It is. I replaced it with a custom class.

I can't seem to find those styles. Were they removed? At least from what I can tell in previewing the Federalist site, there's no styles being assigned aside from the default link styles. We should be able to remove it altogether?

Not in scope here, but @nickttng had brought up that we might consider if marking these up as a list would be the best approach anyways, as in the examples of "Topic" -> "Secondary link" in the [USWDS footer guidance](https://designsystem.digital.gov/components/footer/).